src/components/Navbar.tsx

```diff
@@ -10,20 +28,87 @@ export default function Navbar() {
             height={80}
             width={80}
             alt="Google Dev Group logo"
-            className="mr-4 "
+            className="mr-4"
           />
         </div>
 
         <div className="flex space-x-6 text-white">
-          <a href="#chapters" className="text-body-xl ">
-            Chapters
-          </a>
-          <a href="#guides" className="text-body-xl ">
-            Guides
-          </a>
-          <a href="#pathways" className="text-body-xl ">
-            Pathways
-          </a>
+          <Link
+            href={ROUTES.HOME}
+            className="text-body-xl hover:text-gray-300 transition-colors duration-200"
+          >
+            Home
+          </Link>
+          <Link
+            href={ROUTES.GDG}
+            className="text-body-xl hover:text-gray-300 transition-colors duration-200"
+          >
+            GDG Page
+          </Link>
+          <div className="relative inline-block text-left">
+            <button
+              onClick={() => setIsOpen(!isOpen)}
+              className="inline-flex justify-center items-center w-full text-body-xl text-white hover:text-gray-300 transition-colors duration-200"
+            >
+              Pathways
+              <svg
+                className={`ml-2 -mr-1 h-5 w-5 transform transition-transform duration-200 ${isOpen ? "rotate-180" : ""}`}
+                xmlns="http://www.w3.org/2000/svg"
+                viewBox="0 0 20 20"
+                fill="currentColor"
+                aria-hidden="true"
+              >
+                <path
+                  fillRule="evenodd"
+                  d="M5.293 7.293a1 1 0 011.414 0L10 10.586l3.293-3.293a1 1 0 111.414 1.414l-4 4a1 1 0 01-1.414 0l-4-4a1 1 0 010-1.414z"
+                  clipRule="evenodd"
+                />
+              </svg>
```

Comment:
these can just go in an svg file and be imported as an `<Image>` for optimization